### PR TITLE
Add specs for CreatesMenu interactor.

### PR DESCRIPTION
**Pull request by ugisozols**

### CODE REVIEW

File: core/abc/interactors/creates_menu.rb [hunk 1]
```diff
@@ -1,9 +1,11 @@
-require 'abc/entities/menu/menu'
-require 'abc/entities/menu/menu_entry'
+require File.expand_path('../../entities/menu/menu', __FILE__)
+require File.expand_path('../../entities/menu/menu_entry', __FILE__)
 
 module Abc
   module Interactors
     class CreatesMenu
+      attr_reader :menu
```
Comment (robyurkowski) [inline core/abc/interactors/creates_menu.rb:7]: I like this change.

File: core/abc/interactors/creates_menu.rb [hunk 2]
```diff
@@ -1,9 +1,11 @@
-require 'abc/entities/menu/menu'
-require 'abc/entities/menu/menu_entry'
+require File.expand_path('../../entities/menu/menu', __FILE__)
+require File.expand_path('../../entities/menu/menu_entry', __FILE__)
```
Comment (knewter) [inline core/abc/interactors/creates_menu.rb]: why no require_relative?

Comment (robyurkowski) [inline core/abc/interactors/creates_menu.rb]: Why require relative at all? Just load as above, no? This is pretty explicit, which is nice -- once we move to a relative require, we're stuck with that relativity, which is even harder to find when stuff is moving as frequently as it is now.

Comment (knewter) [inline core/abc/interactors/creates_menu.rb]: You're right.  Thanks :)

Comment (robyurkowski) [inline core/abc/interactors/creates_menu.rb]: Cheers, @knewter! Also, thanks, @ugisozols. Going to patch this one thing and merge.

Comment (parndt) [inline core/abc/interactors/creates_menu.rb]: I preferred the old way.

**Pull request merged**